independent-projects/tools/registry-descriptor/src/test/java/io/quarkus/registry/builder/ExtensionRegistryBuilderTest.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.registry.builder;
+
+import io.quarkus.registry.ExtensionRegistry;
+import io.quarkus.registry.impl.DefaultArtifactResolver;
+import java.io.IOException;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ExtensionRegistryBuilderTest {
+
+    @Test
+    void shouldResolveCamelDependencies() throws IOException {
+        DefaultArtifactResolver resolver = new DefaultArtifactResolver();
+        ExtensionRegistry registry = new ExtensionRegistryBuilder(resolver)
+                .addPlatform("io.quarkus", "quarkus-universe-bom", "1.5.1.Final")
+                .build();
```

Comment:
Is this resolving actual 1.5.1.Final platform release?

Reply:
It uses the DefaultArtifactResolver which does the real thing, so yes :)

Reply:
Could we use the local quarkus-bom instead?